swap_meet/item.py

```diff
@@ -1,2 +1,38 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+
+        #refactor validation of condition
+        self.condition = condition
+        if 0 <= self.condition <= 5:
+            self.condition = float(condition)
+        else:
+            raise ValueError("Condition must range from 0-5.")
+
+    def get_category(self):
+        return f"{self.__class__.__name__}"
+
+    def __str__(self):
+        item_summary = f"An object of type {self.get_category()} with id {self.id}"
+        return item_summary
+
+    def condition_description(self):
```

Comment:
Love the descriptions here :)

Though, you probably now could see that a `dict` might be useful here: for n conditions to check, a `dict` will be O(1) lookup, vs worse case O(n) to fall through all of the `if/else` conditions to the last one.